custom/ewsghana/handler.py

```diff
@@ -8,6 +8,14 @@
 from custom.ilsgateway.tanzania.handlers.notsubmitted import NotSubmittedHandler
 
 
+def looks_like_prod_code(code):
```

Comment:
This is kind of confusing to me. Basically if the code can be coerced into a float, it's a prod code? This might also be of use [is_digit](https://docs.python.org/2/library/stdtypes.html#str.isdigit)